EFCore.VisualBasic/Scaffolding/Internal/VisualBasicModelGenerator.vb

```diff
@@ -9,34 +11,22 @@ Namespace Scaffolding.Internal
     Public Class VisualBasicModelGenerator
         Inherits ModelCodeGenerator
 
-        ''' <summary>
-        '''     This is an internal API that supports the Entity Framework Core infrastructure and not subject to
-        '''     the same compatibility standards as public APIs. It may be changed or removed without notice in
-        '''     any release. You should only use it directly in your code with extreme caution and knowing that
-        '''     doing so can result in application failures when updating to a new Entity Framework Core release.
-        ''' </summary>
-        Public Overridable ReadOnly Property _VBDbContextGenerator As IVisualBasicDbContextGenerator
-
-        ''' <summary>
-        '''     This is an internal API that supports the Entity Framework Core infrastructure and not subject to
-        '''     the same compatibility standards as public APIs. It may be changed or removed without notice in
-        '''     any release. You should only use it directly in your code with extreme caution and knowing that
-        '''     doing so can result in application failures when updating to a new Entity Framework Core release.
-        ''' </summary>
-        Public Overridable ReadOnly Property _VBEntityTypeGenerator As IVisualBasicEntityTypeGenerator
-
+        Private _VBDbContextGenerator As VisualBasicDbContextGenerator
+        Private _VBEntityTypeGenerator As VisualBasicEntityTypeGenerator
```

Comment:
Was there an interest in keeping these properties?

Reply:
Nope